res/controllers/Keyboard-Default.kbd.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Thanks, I wasn't aware of the existence of this windows NSIS uninstall. The Windows installer will be updated in the next push.